pkg/storage-init/storage-init.sh

```diff
@@ -117,7 +117,7 @@ fi
 P3=$(findfs PARTLABEL=P3)
 IMGA=$(findfs PARTLABEL=IMGA)
 IMGB=$(findfs PARTLABEL=IMGB)
-if [ -n "$IMGA" ] && [ -z "$P3" ] && [ -z "$IMGB" ]; then
+if [ -n "$IMGA" ] && [ -z "$P3" ]; then
```

Comment:
What is the rationale of removing the checking for a missing IMGB?

Reply:
@rene it is still there but checked individually later. For evaluation image IMGB will never be empty

Reply:
no, we don't and we do not do it. we recover only if IMGB is missing which is the case for VM images only and not the case for evaluation. For the same reason I do not check for IMGC at all.  But P3 may be missing e.g. for live image so we need to recreate it for any platform. this is exactly why I moved this condition from upper if -- now it can handle evaluation as well